addons/website/views/new_page_template_templates.xml

```diff
@@ -1,6 +1,176 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
 
+<!-- This template is used to obtain the order and the translations of all
+group names.
+It is not done in python (eg: a method returning a list of translated terms,
+overridden by modules), because:
+- it needs to be simple for designers and community: only write XML in themes
+- python is ignored when importing modules in Odoo, which is a common flow for
+  themes
+-->
+<template id="new_page_template_groups">
+    <div id="basic">Basic</div>
+    <div id="about">About</div>
+    <div id="landing">Landing Pages</div>
+    <div id="gallery">Gallery</div>
+    <div id="services">Services</div>
+    <div id="pricing">Pricing Plans</div>
+    <div id="team">Team</div>
+</template>
+
+<!-- Specific templates are generated from the manifest files:
+    If the manifest defines:
+    'new_page_template': {
+        'somekind': {
+            'somespecific': ['s_cover'... list of snippets],
+        },
+    }
+    A template named: new_page_template_sections_somekind_somespecific is generated.
+    Its arch contains t-snippet-calls to each listed snippet, all nested in a div#wrap.
+-->
+
+<!-- Custom blocks -->
+<template id="s_text_block_title_blank" inherit_id="website.s_text_block" primary="True">
```

Comment:
Followup of my previous comments in this review about removing the fact that "blank" is simulating being a regular preview, it would also allow to delete this template, no?
And also remove it from the manifest.
Only the JS would add a programmatically created "blank" entry as a first preview of the first page?